FCWTAPI/CWTObject.cs

```diff
@@ -86,6 +86,20 @@ public void SplitRealAndImaginary(CWTComponent comp, out double[,]? realCwt, out
                     break;
             }
         }
+        public double[,]? GetComponent(CWTComponent comp, CWTOutput cwtOutput)
```

Comment:
If the only function of this method to return Real, Imaginary or Both as as a single array, wouldn't it make more sense to just have a method to get both components in a single array since SplitRealAndImaginary already gets a double[ , ] corresponding to the real and imaginary components?